### ISSUE TITLE
Section background when set to "dynamic" > "featured image": does not respond to background size or background position settings

### ISSUE THREAD

**Issue by portrayaloflife** [elementor/elementor#13373]
## Prerequisites

- [x] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [x] The issue still exists against the latest stable version of Elementor.

When setting a section background and using "dynamic" > "featured image". The image on the backend appears correct when setting background size or background position settings but on the front end those settings don't take. 

Create a section, choose a dynamic background image based on the featured image of the post. Set a background position such as "top right" and a size of "cover" > update and load page in front end to see those image setting not working.

## Isolating the problem

- [x] This bug happens with only Elementor plugin active (and Elementor Pro).
- [x] This bug happens with a default WordPress theme active.
- [x] I can reproduce this bug consistently using the steps above.

## Environment

<details>
<summary>System Info</summary>
```
<!-- For bug reports, let us know about your system environment: Copy and paste the system info report from Elementor => System info, and paste it here or in http://pastebin.com/ -->
```
</details>

**Comment by portrayaloflife** [comment]
?

**Comment by portrayaloflife** [comment]
@shilo-ey

**Comment by sermalefico** [comment]
Yes, dynamic images lose style position and size every time you save page or change featured image. This bug is when you assign a template to a page and put a dynamic image featured background image  in a column

**Comment by portrayaloflife** [comment]
@shilo-ey can you address?

**Comment by spyrosntanos** [comment]
Same problem. Any solution?

**Comment by fabienlege** [comment]
One more a time, a very recurring bug, reported more than two year ago and always nothing do by Elementor team. This plugin is really abandoned.

**Comment by rvisima** [comment]
Bug still occurring three years later. Can we get some attention on this?

**Comment by sylvastudio** [comment]
Bug still exists on Elementor Version 3.26.5

**Comment by portrayaloflife** [comment]
Our quick fix has been setting a custom class to the container and setting background position and cover in CSS
